### PR TITLE
Add values to food category

### PR DESCRIPTION
**Pull request by satyawikananda**

### CODE REVIEW

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
The tests are failing because you can't add values that are more than 1 word. Please remove these.
Also, the CONTRIBUTING.md file asks that you contribute _at least_ 10 values. Thanks!

**Comment by BalgopalPatro** [comment]
@satyawikananda just remove the data which has " "(SPACE)

File: qual_id/categories/food.py [hunk 1]
```diff
@@ -20,5 +20,26 @@ def get_values(self):
         'potato',
         'kiwi',
         'strawberry',
+        'pie',
+        'pizza',
+        'burger',
+        'kebab',
+        'noddle',
+        'egg benedict',
```
Comment (gabrielbarker) [inline qual_id/categories/food.py:28]:
```suggestion
        'bagel',
```

File: qual_id/categories/food.py [hunk 2]
```diff
@@ -20,5 +20,26 @@ def get_values(self):
...
+        'fried rice',
```
Comment (gabrielbarker) [inline qual_id/categories/food.py:29]:
```suggestion
        'croissant',
```

File: qual_id/categories/food.py [hunk 3]
```diff
@@ -20,5 +20,26 @@ def get_values(self):
...
+        'spaghetti',
+        'cabbage',
+        'carpaccio',
+        'sushi',
+        'ramen',
+        'onigiri',
+        'udon',
+        'sashimi',
+        'curry',
+        'fried chicken',
```
Comment (gabrielbarker) [inline qual_id/categories/food.py:40]:
```suggestion
        'chicken',
```

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
I've made some suggestions that don't have ' ' characters, please accept them and I will merge 🙂

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
*

**Review by gabrielbarker — COMMENTED** [review]
*batch*

**Pull request closed without merging**